src/Security/Handler/AclSecurityHandlerInterface.php

```diff
@@ -49,7 +50,7 @@ public function getObjectPermissions();
     /**
      * Get the ACL for the passed object identity.
      *
-     * @return AclInterface|null or NULL if not found
+     * @return MutableAclInterface|null or NULL if not found
```

Comment:
The phpdoc comment is weird

`MutableAclInterface|null or Null`

IMHO we could remove the `or NULL if not found`